### PR TITLE
[BugFix] Adding a minimum quantity to one by one input

### PR DESCRIPTION
**Pull request by JooLuiz**
#### What does this PR do? \*

Sets a minimum quantity for the quantity input in the Quick Order One By One

#### How to test it? \*

Search for a product in the Quick Order one by one and try to put a quantity that is less than 1.

#### Describe alternatives you've considered, if any. \*

None

#### Related to / Depends on \*

None

### CODE REVIEW

**Comment by JooLuiz** [comment]
> Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! 🤖
> 
> Please select which version do you want to release:
> 
> * [X]  Patch (backwards-compatible bug fixes)
> * [ ]  Minor (backwards-compatible functionality)
> * [ ]  Major (incompatible API changes)
> 
> And then you just need to merge your PR when you are ready! There is **no need** to create a release commit/tag.
> 
> * [ ]  No thanks, I would rather do it manually 😞

**Comment by JooLuiz** [comment]
> Beep boop 🤖
> 
> I noticed you didn't make any changes at the `docs/` folder
> 
> * [X]  There's nothing new to document 🤔
> * [ ]  I'll do it later 😞
> 
> In order to keep track, I'll create an issue if you decide now is not a good time
> 
> * [ ]  I just updated 🎉🎉

**Review by wender — CHANGES_REQUESTED** [review]
Can you please update the Changelog file?

**Comment by JooLuiz** [comment]
@wender changelog file updated! Thanks a lot!

**Comment by JooLuiz** [comment]
hi @wender 
I've noticed that there is now two more bots.
They are saying that the PR has an error, but doesn't say where or what to do to solve it.
Could you please help me understand these issues? Or is the code okei?

**Comment by JooLuiz** [comment]
Olá @wender ! Poderia revisar o commit para mim?

File: react/AutocompleteBlock.tsx [hunk 1]
```diff
@@ -410,12 +410,15 @@ const AutocompleteBlock: FunctionComponent<any & WrappedComponentProps> = ({
...
                       size="small"
                       type="number"
+                      min="1"
                       step={unitMultiplier}
                       onChange={(e: any) => {
-                        setState({
-                          ...state,
-                          quantitySelected: e.target.value,
-                        })
+                        if (e.target.value > 0){
```
Comment (aurorahey) [inline react/AutocompleteBlock.tsx:416]: Could you please fix this Line error and rebase the CHANGELOG please?

**Review by aurorahey — CHANGES_REQUESTED** [review]

File: react/AutocompleteBlock.tsx (hunk 1 above)
Comment (JooLuiz) [reply react/AutocompleteBlock.tsx:416]: Also, I resolved the conflict of changelog! thanks for the warning!

Comment (aurorahey) [reply react/AutocompleteBlock.tsx:416]: Thank you for update the changeling, it is the Line error from the PR check. You can see the detail from this link [https://github.com/vtex-apps/quickorder/runs/5012812236?check_suite_focus=true](url)

Comment (JooLuiz) [reply react/AutocompleteBlock.tsx]: Thanks @aurorahey ! I just changed the line and added the changed required! it was a space missing hahah

**Review by aurorahey — APPROVED** [review]

**Review by wender — APPROVED** [review]

**Review by whc105 — APPROVED** [review]

**Pull request merged**